GitUI/UserControls/FileStatusDiffCalculator.FileStatusDiffCalculatorInfo.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using GitUIPluginInterfaces;
+
+namespace GitUI
+{
+    partial class FileStatusDiffCalculator
+    {
+        private struct FileStatusDiffCalculatorInfo
+        {
+            public IReadOnlyList<GitRevision> Revisions { get; set; }
+            public Func<ObjectId?, string>? describeRevision;
+            public Func<GitRevision, GitRevision?>? getActualRevision;
```

Comment:
❔ ditto (--> property)

Reply:
Moved the properties to FileStatusDiffCalculator.cs